docs/CHANGES.md

```diff
@@ -3,6 +3,7 @@
 ## 3.0.1 (Unreleased)
 
 - #585 Duplicate contact created due to JID case sensivity [saganshul]
+- #628 Fixes typing status for typing from different devices. [smitbose]
```

Comment:
This is not descriptive of what was done. It's not a bugfix, you simply changed the message that gets displayed. Also please put it after saganshul's message.